models/intermediate/int_quickbooks__retained_earnings.sql

```diff
@@ -4,19 +4,42 @@ with general_ledger_balances as (
     from {{ ref('int_quickbooks__general_ledger_balances') }}
 ),
 
-net_income_loss as (
+revenue_starter as (
 
     select
         period_first_day,
         source_relation,
-        sum(case when account_class = 'Revenue' then period_net_change else 0 end) as revenue_net_change,
-        sum(case when account_class = 'Revenue' then period_net_converted_change else 0 end) as revenue_net_converted_change,
-        sum(case when account_class = 'Expense' then period_net_change else 0 end) as expense_net_change,
-        sum(case when account_class = 'Expense' then period_net_converted_change else 0 end) as expense_net_converted_change
+        sum(period_net_change) as revenue_net_change,
+        sum(period_net_converted_change) as revenue_net_converted_change
     from general_ledger_balances
+    
+    where account_class = 'Revenue'
+
```

Comment:
Can you explain these changes in the changelog? It's just not obviously related to the created/update_at if it's part of those changes.

Reply:
Got it. I'll leave it to your judgement!